integrations/intercom/src/index.ts

```diff
@@ -211,17 +231,21 @@ const integration = new Integration({
       },
     })
 
-    for (const part of conversation_parts) {
+    const createInBotpressBot = async (intercomMessage: IntercomMessage) => {
```

Comment:
ok changed, I added a comment instead to explain because I think the whole (bot /integration) concept can be a bit confusing at first and this file has two clients, both which create messages.